lib/Service/Appointments/MailService.php

```diff
@@ -144,11 +149,77 @@ public function sendConfirmationEmail(Booking $booking, AppointmentConfig $confi
 		}
 	}
 
+	/**
+	 * @param Booking $booking
+	 * @param AppointmentConfig $config
+	 * @throws ServiceException
+	 */
+	public function sendBookingInformationEmail(Booking $booking, AppointmentConfig $config, string $calendar): void {
+		$user = $this->userManager->get($config->getUserId());
+
+		if ($user === null) {
+			throw new ServiceException('Could not find organizer');
+		}
+
+		$fromEmail = $user->getEMailAddress();
+		$fromName = $user->getDisplayName();
+
+		$sys = $this->getSysEmail();
+		$message = $this->mailer->createMessage()
+			->setFrom([$sys => $fromName])
+			->setTo([$booking->getEmail() => $booking->getDisplayName()])
+			->setReplyTo([$fromEmail => $fromName]);
+
+
+		$template = $this->mailer->createEMailTemplate('calendar.confirmAppointment');
+		$template->addHeader();
+
+		//Subject
+		$subject = $this->l10n->t('Your appointment "%s" with %s has been accepted', [$config->getName(), $user->getDisplayName()]);
+		$template->setSubject($subject);
+
+		// Heading
+		$summary = $this->l10n->t('Dear %s, your booking has been accepted.', [$booking->getDisplayName()]);
+		$template->addHeading($summary);
+
+		$template->addBodyListItem($user->getDisplayName(), 'Appointment with:');
+		if (!empty($config->getDescription())) {
+			$template->addBodyListItem($config->getDescription(), 'Description:');
+		}
+
+		// Create Booking overview
+		$this->addBulletList($template, $this->l10n, $booking, $config->getLocation());
+
+		$bodyText = $this->l10n->t('If you wish to cancel the appointment after all, please contact your organizer.');
```

Comment:
Maybe add something like « by replying to this email or heading to their profile page » (first part should check if `$fromEmail` isn't `null`).

Reply:
from email is sys email. but I could add a reply to header.

Reply:
I meant when the organizer has no email set, but I had forgotten that can't happen. :)